app/src/main/java/com/github/libretube/api/obj/Streams.kt

```diff
@@ -35,40 +36,33 @@ data class Streams(
     val uploaderSubscriberCount: Long = 0,
     val previewFrames: List<PreviewFrames> = emptyList()
 ) {
-    fun toDownloadItems(
-        videoId: String,
-        fileName: String,
-        videoFormat: String?,
-        videoQuality: String?,
-        audioFormat: String?,
-        audioQuality: String?,
-        subtitleCode: String?
-    ): List<DownloadItem> {
+    fun toDownloadItems(downloadData: DownloadData): List<DownloadItem> {
+        val (id, name, videoFormat, videoQuality, audioFormat, audioQuality, subCode) = downloadData
```

Comment:
Why are we calling the language of the subtitle `subCode` instead of `subtitleCode`? Wouldn't the latter be more self-explaining?

Reply:
That was to avoid lint issues due to the line length.